inference-engine/src/mkldnn_plugin/nodes/non_max_suppression.cpp

```diff
@@ -250,7 +250,9 @@ class NonMaxSuppressionImpl: public ExtLayerBase {
             }
             numFiltBox[batch_idx][class_idx] = fb.size();
             size_t offset = batch_idx*num_classes*max_output_boxes_per_class + class_idx*max_output_boxes_per_class;
-            cpu_memcpy(filtBoxes.data() + offset, fb.data(), fb.size() * sizeof(filteredBoxes));
+            for (size_t i = 0; i < fb.size(); i++) {
```

Comment:
Why did you change memcpy to cycle?

Reply:
Implementation with ```cpu_memcpy``` work incorrect in CentOS7